### PR TITLE
[11.0][MIG] hr_timesheet_task_required migration

### PR DESCRIPTION
**Pull request by benwillig**

### CODE REVIEW

File: hr_timesheet_task_required/README.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (feketemihai) [inline hr_timesheet_task_required/README.rst]: @benwillig Please update readme, png images, runbot version

**Review by feketemihai — CHANGES_REQUESTED** [review]
Rest is fine :+1:

**Review by feketemihai — APPROVED** [review]

**Comment by benwillig** [comment]
@feketemihai Done

**Comment by feketemihai** [comment]
@benwillig Thanks for the update, i already approved since it was not a blocker...

**Review by hbrunn — APPROVED** [review]

**Pull request merged**